recipes/perl-uri/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "perl-uri" %}
+{% set version = "5.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/O/OA/OALDERS/URI-5.17.tar.gz
+  sha256: 5f7e42b769cb27499113cfae4b786c37d49e7c7d32dbb469602cd808308568f8
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - make
+  host:
+    - perl 5.32.1
+    - perl-business-isbn
+    - perl-carp
+    - perl-constant
+    - perl-data-dumper
+    - perl-encode
+    - perl-exporter
+    - perl-extutils-makemaker
+    - perl-file-path
+    - perl-file-temp
+    - perl-mime-base64
+    - perl-parent
+    - perl-test
+    - perl-test-harness
+    - perl-test-needs
+    - perl-test-warnings
+    - perl-test-fatal
+  run:
+    - perl >=5.32.1,<5.33.0a0 *_perl5
+    - perl-business-isbn
+    - perl-carp
+    - perl-constant
+    - perl-data-dumper
+    - perl-encode
+    - perl-exporter
+    - perl-mime-base64
+    - perl-parent
+
+test:
+  imports:
+    - URI
+
+about:
+  home: https://github.com/libwww-perl/URI
+  summary: 'Uniform Resource Identifiers (absolute and relative)'
+  description: |
+    This module implements the URI class. Objects of this class represent "Uniform Resource Identifier references" as specified in RFC 2396 (and updated by RFC 2732).
+  license: GPL-1.0-or-later
+  license_file: LICENSE
+  dev_url: https://github.com/libwww-perl/URI
+
+extra:
+  recipe-maintainers:
```

Comment:
It is your choice as recipe author, but if you added `conda-forge/perl-packagers` as additional recipe maintainer, we could help to keep this recipe up and running in case of problems ;-)